### DISCUSSION TITLE
Question: How to go full screen?

### DISCUSSION THREAD

**Discussion by JacobDoes** [FunkinCrew/Funkin#1568] [Q&A]
I want to record a video on Friday Night Funkin' and I don't know how to go full screen. I am using Newgrounds, not itch.io.

**Comment by ghost** [comment]
During the credits or logo screens, press f once.

**Comment by ghost** [comment]
Press Alt Enter

**Comment by ghost** [comment]
f

**Comment by ghost** [comment]
f

**Comment by Windows1Door2Daniel** [comment]
> Press Alt Enter

Thats only for downloaded version

**Comment by ACardboardBox1** [comment]
Press F.
